api/app/reviews/repository.py

```diff
@@ -213,6 +214,45 @@ def get_review_complete_timeseries_by_event(event_id):
                         .all())
         return timeseries
 
+    @staticmethod
+    def get_candidate_response_ids(event_id, reviewer_user_id, reviews_required):
+        candidate_responses = db.session.query(Response.id) \
+            .filter(Response.user_id != reviewer_user_id,
+                    Response.is_submitted == True,
+                    Response.is_withdrawn == False) \
+            .join(ApplicationForm, Response.application_form_id == ApplicationForm.id) \
+            .filter(ApplicationForm.event_id == event_id) \
+            .outerjoin(ResponseReviewer, Response.id == ResponseReviewer.response_id) \
+            .group_by(Response.id) \
+            .having(func.count(ResponseReviewer.reviewer_user_id) < reviews_required) \
+            .all()
+        return candidate_responses
+
+    @staticmethod
+    def get_already_assigned(reviewer_user_id):
+        already_assigned = db.session.query(ResponseReviewer.response_id) \
+            .filter(ResponseReviewer.reviewer_user_id == reviewer_user_id) \
+            .all()
+        return already_assigned
+
+    @staticmethod
+    def get_reviewer(event_id, user_id):

```

Comment:
I don't think this function really belongs in this repository because it's doing more than just querying the database (by calling is_reviewer on the AppUser object). On line 419 of api.py, rather use the user_repository to get the user and call is_reviewer there instead.